testing/matrix-sdk-test/src/event_factory.rs

```diff
@@ -360,6 +361,32 @@ impl EventBuilder<RoomMessageEventContent> {
     }
 }
 
+impl EventBuilder<UnstablePollStartEventContent> {
+    /// Adds a reply relation to the current event.
+    pub fn reply_to(mut self, event_id: &EventId) -> Self {
+        if let UnstablePollStartEventContent::New(content) = &mut self.content {
+            content.relates_to = Some(RelationWithoutReplacement::Reply {
+                in_reply_to: InReplyTo::new(event_id.to_owned()),
+            });
+
+            self.content = UnstablePollStartEventContent::New(content.to_owned());
```

Comment:
Is this required? We should've modified the `content` in place above, because of the `&mut self.content`…

Reply:
Nop, that's just me botching the language 🙈